src/_stylesheets/_swatch.scss

```diff
@@ -0,0 +1,62 @@
+.app-swatch {
+  $sample-size: 2.5em;
+
+  display: flex;
```

Comment:
Give we'll be using CSS grid for the layout anyways, we could use it here as well, which would allow to remove the `app-swatch__label` div 😊 

<img width="1462" height="798" alt="Screenshot 2025-08-04 at 13 11 12" src="https://github.com/user-attachments/assets/02b5b97e-c7b1-4313-aaca-a3c01c6257f7" />